crates/vm2/src/single_instruction_test/callframe.rs

```diff
@@ -42,7 +42,6 @@ impl<'a, T: Tracer, W: World<T>> Arbitrary<'a> for Callframe<T, W> {
             sp: u.arbitrary()?,
             // It is assumed that it is always possible to add the stipend
             gas: u.int_in_range(0..=u32::MAX - EVM_SIMULATOR_STIPEND)?,
```

Comment:
I think this no longer needs to be constrained, as nothing adds gas anymore. The constant can be deleted as well.